### ISSUE TITLE
add former and next buttons when browsing through members and parcels

### ISSUE THREAD

**Issue by kermie** [parcella-garden/parcella#202]
As a Parcella user, I want to have a former and next button on the top of details pages /members/{UUID} and /parcels/{UUID}. 

When working on members and parcels manually and one after another, it feels clumsy to go onto a member or parcel detail page, change something, go back to the list at /members or parcels/, find the next candidate to adapt and step into the according detail page again.

wanted behaviour: add a <former> and <next> button to each detail page in members and parcels so I can browse smoothly to the next (or former) list entry.

**Comment by kermie** [comment]
Fixed in 313aecf: Previous/Next buttons on /members/{id} and /parcels/{id}, walking the same filtered/ordered list you were browsing (not the full table). Also added a Ctrl+Left/Ctrl+Right keyboard shortcut. Details, including why the first keyboard-shortcut attempt (Alt+Arrow) silently failed in some browsers, are in ADR 0075.
